dm/worker/worker_test.go

```diff
@@ -161,7 +162,7 @@ func (t *testServer) TestTaskAutoResume(c *C) {
 	// start task
 	cli := t.createClient(c, fmt.Sprintf("127.0.0.1:%d", port))
 	subtaskCfgBytes, err := ioutil.ReadFile("./subtask.toml")
-	_, err = cli.StartSubTask(context.Background(), &pb.StartSubTaskRequest{Task: string(subtaskCfgBytes)})
+	_, err = cli.StartSubTask(context.Background(), &pb.StartSubTaskRequest{Task: strings.Replace(string(subtaskCfgBytes), "#extra-args", "extra-args", 1)})
```

Comment:
why update this?

Reply:
Because this test uses `github.com/pingcap/dm/dm/worker/subtask.toml` as input and in this file `extra-args` is commented. Without this mydumper will try to connect to the DB and generate `extra-args` which will cause failure.

Reply:
ok, can add a comment for this